src/parsing/lpParser.mly

```diff
@@ -378,31 +377,32 @@ sterm:
 term:
   | t=sterm { t }
   | t=term ARROW u=term { make_pos $sloc (P_Impl(t, u)) }
-  // Quantifier
-  | BACKQUOTE q=term_ident b=binder
-      {
-        let bder = Pos.make b.pos (P_Abst(fst b.elt, snd b.elt)) in
-        make_pos $sloc (P_Appl(q, bder))
-      }
-  | PI b=binder { make_pos $sloc (P_Prod(fst b.elt, snd b.elt)) }
-  | LAMBDA b=binder { make_pos $sloc (P_Abst(fst b.elt, snd b.elt)) }
-  | LET x=ident a=arg_list* b=preceded(COLON, term)? ASSIGN t=term IN u=term
-      { make_pos $sloc (P_LLet(fst x, a, b, t, u)) }
+  | BACKQUOTE q=term_id b=binder
+    {
+      let b = make_pos $loc(b) (P_Abst(fst b, snd b)) in
+      make_pos $sloc (P_Appl(q, b))
+    }
+//| i=ID_QUANT b=binder
+//  {
+//    let q = make_pos $loc(i) (P_Iden(qid_of_mod $loc(i) (fst i), snd i)) in
+//    let b = make_pos $loc(b) (P_Abst(fst b, snd b)) in
+//    make_pos $sloc (P_Appl(q, b))
+//  }
```

Comment:
Do you want to keep these comments as well?

Reply:
No, I will remove them before merging (if I do not remember ;-))